service/kafka-plugin/src/main/java/com/solace/maas/ep/runtime/agent/plugin/kafka/KafkaApplication.java

```diff
@@ -0,0 +1,11 @@
+package com.solace.maas.ep.runtime.agent.plugin.kafka;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
```

Comment:
Do we need plugin applications? I thought a standalone plugin would do nothing.

Reply:
It needs a main class for the Spring Loader.